### ISSUE TITLE
batchnorm

### ISSUE THREAD

**Issue by yja1** [CaoWGG/Mobilenetv2-CenterNet#2]
why set different param batchnorm, in base layer:
nn.BatchNorm2d(out_planes)

but in IDAUp and MobileNetUp:
nn.BatchNorm2d(out_dim,eps=0.001,momentum=0.1)

**Comment by CaoWGG** [comment]
@yja1 
same as centerface
